crowdsourcing/crowdsourcing_tutorial.py

```diff
@@ -143,7 +148,7 @@ def get_worker_labeling_function(worker_id, f):
 # %%
 from snorkel.labeling.analysis import LFAnalysis
 
-LFAnalysis(L_dev, lfs).lf_summary(Y_dev).head(10)
+LFAnalysis(L_dev, lfs).lf_summary(Y_dev).head(8)
```

Comment:
? seems arbitrary

Reply:
The 9th entry is a worker who has no labels in the dev set (but does have labels in the train set). This confused some people in the feedback session. The alternative is to add a comment explaining this, but it seemed like it would be less distracting to just not show that entry.

Reply:
I see—super nit, but maybe we can just show `.head(5)`? Seems less arbitrary than 8

Reply:
Done